spec/features/guest_navigation_spec.rb

```diff
@@ -121,19 +121,25 @@
   end
 
   context 'on the "/cats" page' do
-    xit 'shows a proper list of cats'do
-      # 045
-      # As a Guest
-      # When I visit '/cats'
-      # Then I see a list of all available cats each with picture and name
+    it 'shows a proper list of cats'do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.